docs/_quarto.yaml

```diff
@@ -130,79 +123,73 @@ quartodoc:
     include_attributes: true
 
   sections:
-    - title: Cloud Services
-      desc: Cloud Interaction Services
+    - title: Catalog
+      desc: PLACEHOLDER CATEGORY DESCRIPTION
```

Comment:
Make sure you update this with a category. If there's one in TileDB-Documentation, feel free to use that. However, I'm not sure if it's needed by Monday. I'll let someone else decide that.